pages/test_analytics/importing_json.md.erb

```diff
@@ -239,4 +239,23 @@ The JSON format is not yet documented, but the following example is what is prod
 
 ## Environment variables
 
-<%= render_markdown 'test-analytics/_env_variables_ta' %>
```

Comment:
This would be **without** the trailing underscore. :/

Reply:
I guess it's so Rails doesn't think it is a file it needs to care about otherwise.

Reply:
Yeah, I've been meaning to delete it before committing, but forgot to, @plaindocs